nginx.conf

```diff
@@ -5,6 +5,14 @@ server {
     root /usr/share/nginx/html;
     index index.html;
 
+    add_header X-Frame-Options SAMEORIGIN;
+    add_header X-Content-Type-Options nosniff;
+    add_header Referrer-Policy strict-origin-when-cross-origin;
+    add_header X-Permitted-Cross-Domain-Policies none;
+    add_header Cross-Origin-Resource-Policy same-site;
+    add_header Cross-Origin-Opener-Policy same-origin;
+    add_header Permissions-Policy 'accelerometer=(), autoplay=(self), camera=(), display-capture=(), document-domain=(), encrypted-media=(), fullscreen=(), gamepad=(), geolocation=(), gyroscope=(), magnetometer=(), microphone=(), picture-in-picture=(), usb=(), screen-wake-lock=(), xr-spatial-tracking=()';
```

Comment:
Some tools may require camera (https://sharevb-it-tools.vercel.app/camera-recorder) or geolocation (https://sharevb-it-tools.vercel.app/geo-distance-calculator) or full-screen (https://sharevb-it-tools.vercel.app/time-alarm) or microphone (https://sharevb-it-tools.vercel.app/mic-tester)

Reply:
Yes that's true, I missed some things. I will fix this and retest

Reply:
Actually, I removed the `Permissions-Policy`. There are so many options and I can't be sure which tools actually need a specific permission. If someone later wants to add the header, there is a nice tool which helps crafting the header: https://www.permissionspolicy.com/